wagtail/admin/views/pages/create.py

```diff
@@ -182,13 +182,46 @@ def get_view_draft_message_button(self):
     def get_view_live_message_button(self):
         return messages.button(self.page.url, _("View live"), new_window=False)
 
+    def set_default_privacy_setting(self):
```

Comment:
Question: It looks like this method overrides whatever setting is currently set - is that intentional? It looks like it's currently being called with every save, rather than just initial creation, but I may be wrong.

Reply:
I think this class (CreateView) is only used for creation. After the page is created I think the EditView is used so it should only be the initial creation.